en/docs/apis/advanced-topics/connect-to-an-external-identity-provider.md

```diff
@@ -0,0 +1,146 @@
+
+3. Go to the **API Management** tab and click **Identity Providers**. This displays the existing identity providers in your organization if you have already added any.
+
+    ![Identity provider list](../../assets/img/apis/advanced-topics/external-identity-providers/identity-provider-list.png){.cInlineImage-full}
+
+4. Click **+ Add Provider**. This displays the identity providers supported by Choreo.
+
+    ![Identity providers](../../assets/img/apis/advanced-topics/external-identity-providers/identity-providers.png){.cInlineImage-full}
+
+5. Click on the IdP you want to add. This displays a form where you must enter details to set up the IdP. <br/> This example walks you through the steps to add **Okta** as the IdP. The details you need to fill in are similar for other IdPs as well.
+
+6. Enter appropriate values for each of the fields and click Next. The following table describes each field in detail: 
+
+    <table>
+    <tr class="header">
+    <th><b>Field</b></th>
+    <th><b>Description</b></th>
+    </tr>
+    <tr class="odd">
+    <td>**Name**</td>
+    <td>The name of the IdP. You cannot modify the value after you add the IdP.</td>
+    </tr>
+    <tr class="even">
+    <td>**Description**</td>
+    <td>A brief description of the IdP.</td>
+    </tr>
+    <tr class="odd">
+    <td>**Allowed Token Audience**</td>
+    <td><b>Okta:</b><br>The audience of the authorization server for which the access token is necessary.<br>
+        You can update this default value via the Okta authorization server by navigating to the following path:<br>
+        Security → API → Authorization Server → Your Authorization Server → Settings → Audience<br><br>
+        <b>Microsoft:</b><br>The identifier that identifies the intended recipient of the token.
+This value should be the application ID URI of the application in Azure Active Directory (Azure AD) for which the token is requested.
+        <br><br>
+        <b>Auth0:</b><br>The unique API identifier to use as the audience parameter in the authorization call.<br>
+         You can update this value in the Auth0 server by navigating to the following path:<br>
+         Applications → APIs → Your API → General Settings → Identifier<br>
+    </td>
+    </tr>
+    <tr class="even">
+    <td>**Well-Known URL**</td>
+    <td><p>The OpenID Connect discovery endpoint URL. This returns the metadata related to the OpenID Provider's configuration.</p>
+    </td>
+    </tr>
+    </table>
+
+     ![Add an identity provider](../../assets/img/apis/advanced-topics/external-identity-providers/add-an-identity-provider.png){.cInlineImage-half} 
+
+     When you specify the **Well-Known URL**, the values for the **Issuer**, **Token Endpoint**, and **JWKS Endpoint** fields are filled in automatically in the next screen. Depending on your requirement, you can either keep the values as it is or manually change the values. The following table describes each field in detail:
```

Comment:
How about .. - > When you specify the **Well-Known URL**,  you will find that the **Issuer**, **Token Endpoint**, and **JWKS Endpoint** gets auto-populated. You can either choose to keep the values as it is or manually change them.

Reply:
Changed to "When you specify the **Well-Known URL**, the values for **Issuer**, **Token Endpoint**, and **JWKS Endpoint** get autopopulated. You can either choose to keep the values or manually change them."

going with "autopopulated" instead of "auto-populated" based on https://developers.google.com/style/word-list?hl=en#autopopulate